cmds/cmd_utils.py

```diff
@@ -103,6 +110,24 @@ async def avatar_function(message, args, client, **kwargs):
     await message.channel.send(embed=embed)
 
 
+async def embed(message, args, client, **kwargs):
+    desc = " ".join(args)
+
+    if desc == "":
+        await message.channel.send("Please provide a message to send")
+        return
+
+    desc = desc.replace('\\n', '\n')
+
+    if len(desc) > 2048:
+        message = message[:2047]
```

Comment:
While it is good practice to account for general errors, this is not a general error that could possibly happen, the discord max message length is 2000 exactly, so there is no state in which the length would be over 2048, and this is just adding overhead to the processing